src/manager.js

```diff
@@ -0,0 +1,30 @@
+class Manager {
+  constructor() {
+    this.list = []
+    this.idCounter = 0
+  }
+
+  assignId() {
+    this.list[this.list.length - 1].id = this.idCounter
+    return this.list[this.list.length - 1].id
+  }
+
+  increaseIdCount() {
+    this.idCounter += 1
+    return this.idCounter
+  }
```

Comment:
for create new id you don't need to create a new variable for that. Just when you need id for item you can use:
```javascript
this.list.length + 1
``` 
You can use it for create id for new student of cohort. In this example it will work only for id of manager so you check list of created managers check size of it and after that you just plus 1 and create a new id for new manager.